docs/UserGuide.adoc

```diff
@@ -14,22 +14,22 @@ endif::[]
 
 :repoURL: https://github.com/CS2103JAN2018-W14-B1/main/
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team W14-B1`      Since: `Jan 2018`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+TeachConnect (TC) is created to help teachers to *better manange their contacts* and *remenber students that they had taught*. TeachConnect is also tailored for teachers who would *prefer to use a desktop app for managing contacts*. Most importantly, it is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, TeachConnect will manange your contacts faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
```

Comment:
*to help teachers -> to help teachers and educational professionals
*manange -> manage
*Jump to the <<Quick Start>> to get started -> remove "the"